packages/next/build/webpack/loaders/next-babel-loader.js

```diff
@@ -187,6 +188,17 @@ module.exports = babelLoader.custom(babel => {
         'next-js-transform-define-instance',
       ])
 
+      if (isPageFile) {
```

Comment:
```suggestion
      if (isPageFile && !isServer) {
```
This is only needed for client pages e.g. the same time as the `page-config-plugin` on line 137 right?

Reply:
IMO we should be consistent and prune them in both versions.